dataline-workers/src/test/java/io/dataline/workers/singer/SingerCheckConnectionWorkerTest.java

```diff
@@ -24,114 +24,98 @@
+import io.dataline.workers.JobStatus;
 import io.dataline.workers.OutputAndStatus;
-import io.dataline.workers.PostgreSQLContainerTestHelper;
 import java.io.IOException;
-import java.sql.Connection;
-import java.sql.DriverManager;
-import java.sql.SQLException;
-import org.junit.jupiter.api.BeforeAll;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.testcontainers.containers.PostgreSQLContainer;
 
 public class SingerCheckConnectionWorkerTest extends BaseWorkerTestCase {
 
-  private PostgreSQLContainer db;
+  private static final JsonNode CREDS = Jsons.jsonNode(ImmutableMap.builder().put("apiKey", "123").build());
 
-  @BeforeAll
-  public void initDb() throws SQLException {
-    db = new PostgreSQLContainer();
-    db.start();
-    Connection con =
-        DriverManager.getConnection(db.getJdbcUrl(), db.getUsername(), db.getPassword());
-    con.createStatement().execute("CREATE TABLE id_and_name (id integer, name VARCHAR(200));");
+  private Path jobRoot;
+  private StandardCheckConnectionInput input;
+  private StandardDiscoverSchemaInput discoverInput;
+  private DiscoverSchemaWorker discoverSchemaWorker;
+
+  @BeforeEach
+  public void setup() throws IOException {
+    jobRoot = Files.createTempDirectory("");
+
+    input = new StandardCheckConnectionInput();
+    input.setConnectionConfiguration(CREDS);
+
+    discoverInput = new StandardDiscoverSchemaInput();
+    discoverInput.setConnectionConfiguration(CREDS);
+
+    discoverSchemaWorker = mock(DiscoverSchemaWorker.class);
   }
 
   @Test
-  public void testNonexistentDb()
-      throws IOException, InvalidCredentialsException {
-    final String jobId = "1";
-    JsonNode fakeDbCreds =
-        PostgreSQLContainerTestHelper.getSingerTapConfig(
-            "user", "pass", "localhost", "postgres", "111111");
-
-    final StandardCheckConnectionInput standardCheckConnectionInput =
-        new StandardCheckConnectionInput();
-    standardCheckConnectionInput.setConnectionConfiguration(fakeDbCreds);
-
-    SingerCheckConnectionWorker worker =
-        new SingerCheckConnectionWorker(Integrations.POSTGRES_TAP.getCheckConnectionImage(), pbf);
-    OutputAndStatus<StandardCheckConnectionOutput> run =
-        worker.run(standardCheckConnectionInput, createJobRoot(jobId));
-
-    assertEquals(FAILED, run.getStatus());
-    assertTrue(run.getOutput().isPresent());
-    assertEquals(StandardCheckConnectionOutput.Status.FAILURE, run.getOutput().get().getStatus());
-    // TODO Once log file locations are accessible externally, also verify the correct error message
-    // in the logs
+  public void testSuccessfulConnection() throws InvalidCredentialsException, InvalidCatalogException {
+    OutputAndStatus<StandardDiscoverSchemaOutput> discoverOutput =
+        new OutputAndStatus<>(JobStatus.SUCCESSFUL, mock(StandardDiscoverSchemaOutput.class));
```

Comment:
can you just create an empty Output instead of mocking: `new StandardDiscoverSchemaOutput()`